.github/workflows/conformance.yml

```diff
@@ -39,6 +39,16 @@ jobs:
             consul-image: "hashicorp/consul:1.12"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
             consul-k8s-version: "v0.47.1"
+          - name: "consul@v1.13 + consul-k8s@v0.47.1"
+            api-gateway-image: "hashicorppreview/consul-api-gateway:0.4-dev"
+            consul-image: "hashicorp/consul:1.13"
+            envoy-image: "envoyproxy/envoy:v1.22-latest"
+            consul-k8s-version: "v0.47.1"
+          - name: "consul@v1.14-dev + consul-k8s@v0.47.1"
+            api-gateway-image: "hashicorppreview/consul-api-gateway:0.4-dev"
+            consul-image: "hashicorppreview/consul:1.14-dev"
+            envoy-image: "envoyproxy/envoy:v1.22-latest"
+            consul-k8s-version: "v0.47.1"
```

Comment:
I think it's helpful to add these, but I imagine we're going to reach a point where 1.14 depends on some not-yet-released change in consul-k8s. At that point we'll just need to consume `consul-k9s-version: "main"`